pagopa/io-app
# chore(IT Wallet): [SIW-716] Add issued by claim for PID

## Short description
This PR introduces the "issued by" claim for PID. 

## List of changes proposed in this pull request
- Adds the `entityConfiguration` object to the `PidResponse`;
- Updates the `getPid` function to also return the `entityConfiguration` object while obtaining the PID. This means that the `entityConfiguration` is being persisted with the PID credential;
- Updates `ItwPidClaimsList.tsx` which now renders the "issued by" claim by using the `entityConfiguration`; 
- Aligns `ItwPidPreviewScreen.tsx` and `ItwPidDetailsScreen.tsx` to the aformentioned `ItwPidClaimsList.tsx` change;

## How to test
Obtain a PID and check its preview and detail. They both should render the "issued by" claim. Please note that the link which can be open by pressing the claim is currrently not working even if it's being provided by the issuer.
